apps/package.json

```diff
@@ -100,6 +102,7 @@
     "grunt-sass": "1.1.0",
     "grunt-strip-code": "0.1.2",
     "grunt-webpack": "^1.0.11",
+    "hammerjs": "^2.0.8",
```

Comment:
This is a newer version which supports detecting fast-tapping as well as swiping, but the upgrade changes a lot of the APIs so thought it might make sense to bring this in on its own for this app to start.